charts/illumidesk/Chart.yaml

```diff
@@ -6,7 +6,7 @@ description: An extention of the JupyterHub chart with additional IllumiDesk res
 icon: https://configs.illumidesk.com/images/illumidesk-80.png
 dependencies:
   - name: jupyterhub
```

Comment:
for the jupyterhub bump, wouldn't we need to update the helm chart version for that? 11.1 is not backwards compatible

Reply:
@Abhi94N yes we should bump the version. I would recommend bumping the version after all merges though to make things cleaner.